# ALL:ALL - Se agrega -lstdc++ a libraries_private en el generador de pkgconfig de meson para que funcione el enlace estatico cruzado sin libreria estandar faltante.
--- libplacebo/src/meson.build
+++ libplacebo/src/meson.build
@@ -291,6 +291,7 @@
   name: proj_name,
   description: 'Reusable library for GPU-accelerated video/image rendering',
   libraries: lib,
+  libraries_private: '-lstdc++',
   version: version,
   variables: pc_vars,
   extra_cflags: extra_cflags,
